src/pages/thankyou.tsx

```diff
@@ -35,8 +35,13 @@ const ThankYou = ({ uuid, emailAddress }: ThankYouProps): ReactElement => (
             within the next 5 minutes for the fares you have just told us about.
         </p>
         <p className="govuk-body">You will be contacted should there be a problem with your upload.</p>
+        <h2 className="govuk-heading-m">Your options</h2>
         <p className="govuk-body">
-            If you would like to provide feedback on this service, please click{' '}
+            You now have two options; you can create another fare using this service or go straight to the Bus Open Data
+            Service (BODS) to upload your NeTEx file.
```

Comment:
We should have the verbs in this text matching the buttons below. 
e.g. Instead of...
```You now have two options; you can create another fare using this service or go straight to the Bus Open Data Service (BODS) to upload your NeTEx file.```, 
we would have 
```You now have two options; you can add another fare using this service, or you can continue to the Bus Open Data Service (BODS) to upload your NeTEx file.```